apis/python/src/tiledb/vector_search/ingestion.py

```diff
@@ -2394,6 +2499,13 @@ def scale_resources(min_resource, max_resource, max_input_size, input_size):
                         raise ValueError(
                             f"The random sampling ran into an issue: num_sampled ({num_sampled}) != training_sample_size ({training_sample_size})"
                         )
+                # Add dependencies for normalization
```

Comment:
Do we need to do something like we do for `random_sample_node` below? What will happen right now if we  have `copy_centroids_uri = None`  and `training_sampling_policy != TrainingSamplingPolicy.RANDOM`? It seems like we won't have anything depend on `normalization_nodes`?

Reply:
Just added a new dependency from the centroid nodes

Reply:
Thanks, looks good to me